tests/topotests/isis_srv6_te_topo1/rt1/step1/show_ipv6_route.ref

```diff
@@ -0,0 +1,246 @@
+{
+    "fc00:0:1::/128": [
```

Comment:
This ULA format looks good, these addresses are not used on the internet (unlike 1.1.1.1 with ipv4 that might be configured on the internet), changing it will involve the redefining the topotest from entirely with recreating the jsons references...